services/docker-compose.local.yml

```diff
@@ -53,6 +53,11 @@ services:
       - "8000"
       - "3010:3000"
 
+  invitations:
+    ports:
+      - "8008:8000"
+      - "3013:3000"
```

Comment:
I think you might have forgotten to update the vscode settings for debugging that guy?